packages/create-redwood-app/template/jest.config.js

```diff
@@ -0,0 +1,9 @@
+// This the monorepo jest config
+// Each side has their configuration setup e.g. in `web/jest.config.js`
+
+// Note that jest configs aren't merged!
+
```

Comment:
Most of our config files don't have comments, but maybe we should change that and start commenting them all (graphql.config.js, prettier.config.js, etc). What do you think?

The change I suggested here was just to remove it unless we also comment `web/jest.config.js`, `api/jest.config.js`, etc..
```suggestion
```

Reply:
We should also include a link to related docs in the comment

Reply:
👍 I'm calling it "project configuration" vs "app configuration" which is the toml